components/ui/ViewAllLeadsButton/ViewAllLeadsButton.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import Image from 'next/image';
+import exportImage from '/assets/images/leadslist-icons/export.png';
+import { useRouter } from 'next/navigation';
+import React from 'react';
+
+function ViewAllLeadsButton() {
+  const router = useRouter();
+  const handleCreateLeadButtonClick = () => {
+    router.push('/leads/');
```

Comment:
Instead of direct  link, we can use the `PAGE_ROUTES.Leads` here. So it will looks like:
```javascript
router.push(PAGE_ROUTES.Leads);
```

Reply:
Done